libs/damap/src/lib/components/dmp/dmp-actions/dmp-actions.component.html

```diff
@@ -5,7 +5,7 @@
     <button
       class="action-button"
```

Comment:
To be consistent, should the font weight also be the same as for other buttons? (goes for all actions buttons)

Reply:
For comparison:

![image](https://github.com/sharedRDM/damap-frontend/assets/72449192/9535aba0-1aad-4ffc-9918-b985359f44dd)
![image](https://github.com/sharedRDM/damap-frontend/assets/72449192/4253e6c6-8bda-47de-9e6c-9c8fce3a0bfb)

Reply:
Yes, i can add this.